releases/evm/2025-01-ITS-v2.1.0.md

```diff
@@ -87,6 +87,12 @@ Migration Steps
 
 We’d also like to migrate minter for all native ITS tokens if they weren’t auto migrated, so we can remove the migration logic in the future. We’ll create scripts to do that in the future since this is not a blocker for the upgrade.
 
+### Verify Upgraded ITS Contracts
+
+```bash
```

Comment:
Link to the verification instructions since some other setup needs to be done but we don't need to duplicate cmds here